docs/docs-beta/docs/dagster-plus/deployment/code-locations.md

```diff
@@ -3,4 +3,237 @@ title: "Code locations"
+```
+
+3. A CICD workflow file that contains the steps for adding your code location. Here is an example workflow file for a Dagster+ Hybrid organization:
+
+```yaml
+variables:
+  DAGSTER_CLOUD_ORGANIZATION: <organinization-name>
+  DAGSTER_PROJECT_DIR: .
+  IMAGE_REGISTRY: <account-id>.dkr.ecr.us-west-2.amazonaws.com/<image-name>
+  IMAGE_TAG: $CI_COMMIT_SHORT_SHA-$CI_PIPELINE_ID
+
+stages:
+  - build
+  - deploy
+
+build:
+  stage: build
+  image: docker:latest
+  services:
+    - docker:dind
+  before_script:
+    # # For Gitlab Container Registry
+    # - echo $CI_JOB_TOKEN | docker login --username $CI_REGISTRY_USER --password-stdin $REGISTRY_URL
+    # # For DockerHub
+    # - echo $DOCKERHUB_TOKEN | docker login --username $DOCKERHUB_USERNAME --password-stdin $REGISTRY_URL
+    # # For AWS Elastic Container Registry (ECR)
+    # - apk add --no-cache curl jq python3 py3-pip
+    # - pip install awscli
+    # - echo $AWS_ECR_PASSWORD | docker login --username AWS --password-stdin $IMAGE_REGISTRY
+    # # For Google Container Registry (GCR)
+    # - echo $GCR_JSON_KEY | docker login --username _json_key --password-stdin $REGISTRY_URL
+  script:
+    - docker build . -t $IMAGE_REGISTRY:$IMAGE_TAG
+    - docker push $IMAGE_REGISTRY:$IMAGE_TAG
+
+deploy:
+  stage: deploy
+  dependencies:
+    - build
+  image: ghcr.io/dagster-io/dagster-cloud-action:0.1.43
+  script:
+    - dagster-cloud deployment add-location --deployment prod --image
+    $IMAGE_REGISTRY:$IMAGE_TAG --location-name quickstart --package-name quickstart
+```
+
+## Adding a new code location to a Git monorepo
+
+Many organizations use a Git monorepo to contain multiple Dagster projects. Here is an example of DagsterLab's own [internal data engineering Git repository](https://github.com/dagster-io/dagster-open-platform). To add a new location to a monorepo, you will create a new directory containing:
+- Your Dagster project
+- For Dagster+ Hybrid, you will typically add a Dockerfile:
+
+```
+FROM python:3.11-slim
+# Add any steps to install project system dependencies like java
+
+WORKDIR /opt/dagster/app
+
+COPY . /opt/dagster/app
+
+# Add steps to install the Python dependencies for your Dagster project
+# into the default Python on PATH
+# For example, this project uses setup.py and we install all dependencies into the Docker container
+# using `pip`.
+
+RUN pip install -e .
+```
+
+The final repository structure might look like this:
+
+```
+README.md
+dagster_cloud.yaml
+/.github
+    ...
+/shared
+    setup.py
+    pyproject.toml
+    /shared
+        __init__.py
+        utilities.py
+/core
+    setup.py
+    pyproject.toml
+    Dockerfile
+    /core
+        definitions.py
+        __init__.py
+        /assets
+/new-code-location
+    setup.py
+    pyproject.toml
+    Dockerfile
+    /new-code-location
+        definitions.py
+        __init__.py
+        /assets
+```
+
+Then you will update the `dagster_cloud.yaml` file in the root of the Git repository:
```

Comment:
Nit - remove `you will` and instead have this be `Then, update the...`

What needs to be updated? Should this instead be `Update the dagster_cloud.yaml with your registry image`?